app/_how-tos/authenticate-consumers-with-session-and-key-auth.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Authenticate Consumers with the Key Auth and Sessions plugins
+content_type: how_to
+description: Authenticate Consumers with key authentication and session cookies.
+related_resources:
+  - text: Authentication
+    url: /authentication/
+
+products:
+    - gateway
+
+entities: 
+  - service
+  - consumer
+  - route
+
+plugins:
+    - key-auth
+    - session
+    - request-termination
+
+tags:
+  - authentication
+  - key-auth
+  - session
+
+tools:
+    - deck
+
+works_on:
+    - on-prem
+    - konnect
+
+prereqs:
+  entities:
+    services:
+        - example-service
+    routes:
+        - example-route
+
+tldr:
+    q: How do I authenticate Consumers with session cookies?
+    a: You can use the Session plugin, along with an authentication plugin like Key Authentication, to authenticate Consumers with session cookies. Configure the authentication plugin with credentials, an anonymous Consumer, and associate it with a Gateway Service. Create a named Consumer with a credential for the authentication plugin, as well as an anonymous Consumer. Configure the Session plugin and associate it with the Gateway Service, and then configure the Request Termination plugin to prevent anonymous access.
```

Comment:
```suggestion
    a: You can use the Session plugin, along with an authentication plugin like Key Authentication, to authenticate Consumers with session cookies. In summary, you need to:
    1. Configure the authentication plugin with credentials, an anonymous Consumer, and associate it with a Gateway Service. 
    2. Create a named Consumer with a credential for the authentication plugin, as well as an anonymous Consumer. 
    3. Configure the Session plugin and associate it with the Gateway Service, then configure the Request Termination plugin to prevent anonymous access.
```
Trying to break up the block of text.